app/src/main/java/org/oppia/app/topic/lessons/TopicLessonsFragment.kt

```diff
@@ -33,7 +40,8 @@ class TopicLessonsFragment : InjectableFragment(), ExpandedChapterListIndexListe
     }
   }
 
-  @Inject lateinit var topicLessonsFragmentPresenter: TopicLessonsFragmentPresenter
+  @Inject
```

Comment:
Please revert this to ` @Inject lateinit var topicLessonsFragmentPresenter: TopicLessonsFragmentPresenter`

Reply:
as we discussed, it is resolved. Thanks